TestCases/hybrid_regression.py

```diff
@@ -646,7 +646,7 @@ def main():
     fsi2d.cfg_dir   = "fea_fsi/WallChannel_2d"
     fsi2d.cfg_file  = "configFSI.cfg"
     fsi2d.test_iter = 4
-    fsi2d.test_vals = [4.000000, 0.000000, -3.768521, -4.159940]
+    fsi2d.test_vals = [4, 0, -3.743230, -4.133462]
```

Comment:
Of course it would be nice here to have more coverage on the screen output vals.